k8s/service_controller/service_controller.go

```diff
@@ -15,65 +12,26 @@ import (
 type Options struct {
 	ConfigPath string
 	Backend    string
+	Host       string
 }
 
 var options Options
 
 func init() {
 	flag.StringVar(&options.ConfigPath, "c", ".", "use '-c' option to specify the config file path")
 	flag.StringVar(&options.Backend, "backend", "mem", "backend to use for storing info. 'mem' for in memory backend (default), 'k8s' for kubernetes based backend")
+	flag.StringVar(&options.Host, "host", "localhost:8001", "the host of the backend to connect to")
```

Comment:
Can you change this back to "8080" to match how it used to be and how Kube defaults?